images/installer/nested-libvirt/README.md

```diff
@@ -0,0 +1,40 @@
+## Image to enable launching a libvirt cluster nested in a GCE instance, for libvirt CI testing.
+
+This image contains [nss_wrapper](https://cwrap.org/nss_wrapper.html) to execute ssh commands as 
+a mock user to interact with a GCE instance from an OpenShift container.
+
+OpenShift containers run with an arbitrary uid, but ssh requires a valid user.  `nss_wrapper`
+allows for the container's uid to be mapped to a user inside of a container.
+
+
```

Comment:
nit: the rest of this repo only uses a single blank line before headers.

Reply:
thanks, fixed. :)